### ISSUE TITLE
Blob files stored first cf path could cause disk full

### ISSUE THREAD

**Issue by zaorangyang** [facebook/rocksdb#8535]
Currently, the Blob files path are forced to the first CF path. In general, when users specify multiple CF paths, the first path could have less space. Forcing Blob data to write to the first path could easily cause the path full and then cause IO errors.
https://github.com/facebook/rocksdb/blob/803a40d4123e8447a4b271ff5bb2e582ab45c6ef/db/blob/blob_file_builder.cc#L148-L176

**Comment by ltamasi** [comment]
You are correct that multiple paths are currently unsupported for blob files. (Note that blob files are not associated with levels, so the logic for placing SSTs files does not readily generalize to blobs.) Since in BlobDB use cases we expect most of the data to be in the blob files, one workaround would be to use a single path, namely the one where you would like the blob files to be stored.

**Comment by zaorangyang** [comment]
> You are correct that multiple paths are currently unsupported for blob files. (Note that blob files are not associated with levels, so the logic for placing SSTs files does not readily generalize to blobs.) Since in BlobDB use cases we expect most of the data to be in the blob files, one workaround would be to use a single path, namely the one where you would like the blob files to be stored.

Thanks for your reply. Maybe it would be better if a configuration parameter about blob file path was introduced. I can try to enhance it.
